cockroachdb/Chart.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 name: cockroachdb
 home: https://www.cockroachlabs.com
-version: 8.0.1
+version: 8.0.2
```

Comment:
Since we're adding a new feature I think we should make this new version `8.1.0`.

Reply:
Done. ✅